cli/src/templates/packages/react-navigation/App.tsx.ejs

```diff
@@ -42,6 +49,12 @@ export default function App() {
 			InterBold: require("@tamagui/font-inter/otf/Inter-Bold.otf"),
 		});
 
+		const colorScheme = useColorScheme();
+		const mergedTheme = useMemo(() => ({
+			...DefaultTheme,
+			dark: colorScheme === 'dark',
+		}), [colorScheme]);
```

Comment:
You'd want to use the `DarkTheme` exported from `@react-navigation/native` for dark mode for proper colors. The default theme has light theme colors so this won't work properly:

```suggestion
		const theme = colorScheme === 'dark' ? DarkTheme : DefaultTheme
```

Same for the other places below.

Reply:
fixed!